### ISSUE TITLE
pretrain

### ISSUE THREAD

**Issue by Muxindawang** [pjreddie/darknet#2215]
I want to ask how the weight of classification is used as the initialization weight of detection. I train classification for four days and get top1:84.5 top5: 94.4, but when I use it as the initialization weight on detection, I get the loss=0.
